### ISSUE TITLE
[dataclasses] I3TimeWindow pretty print (Trac #1964)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1956]
Allow nice printing of the I3TimeWindow.  I'm tired of seeing a memory address in the shovel, instead of the actual start and stop times.

Migrated from https://code.icecube.wisc.edu/ticket/1964
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:57", 
    "description": "Allow nice printing of the I3TimeWindow.  I'm tired of seeing a memory address in the shovel, instead of the actual start and stop times.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067237750774", 
    "component": "combo core", 
    "summary": "[dataclasses] I3TimeWindow pretty print", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2017-03-16T19:12:13", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170330T18:51:12: 
* olivas commented:

> I'm assuming the milestone for this was meant to be set to June 2017.


* olivas changed milestone from "June 2016" to "June 2017"

**Comment by icecube-trac** [comment]
Trac update at 20170330T19:01:43: 
* jbraun changed status from "new" to "closed"
* jbraun changed resolution from "" to "fixed"
* jbraun commented:

> In changeset 154512:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="154512"
> Add pretty print for I3TimeWindow.  Also, add fractional seconds to the I3EventHeader pretty print.  Closes #1956
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:57: 
* olivas changed milestone from "June 2017" to ""
* olivas commented:

> Milestone June 2017 deleted
